pkg/agent/openflow/client.go

```diff
@@ -749,6 +750,20 @@ func (c *client) InstallTraceflowFlows(dataplaneTag uint8) error {
 					Done())
 		}
 	}
+	// Copy Antrea NetworkPolicy drop rules
+	for _, conj := range c.policyCache.List() {
+		for _, flow := range conj.(*policyRuleConjunction).metricFlows {
```

Comment:
Why we check metricFlows?

Reply:
As the comment says, Antrea NetworkPolicy drop rules will be in metricFlows.
We should mock all "dropRuleMetricFlow".
```
		if rule.IsAntreaNetworkPolicyRule() && *rule.Action == secv1alpha1.RuleActionDrop {
			metricFlows = append(metricFlows, c.dropRuleMetricFlow(ruleID, isIngress))
			actionFlows = append(actionFlows, c.conjunctionActionDropFlow(ruleID, ruleTable.GetID(), rule.Priority))
		} else {
			metricFlows = append(metricFlows, c.allowRulesMetricFlows(ruleID, isIngress)...)
			actionFlows = append(actionFlows, c.conjunctionActionFlow(ruleID, ruleTable.GetID(), dropTable.GetNext(), rule.Priority))
		}
		conj.actionFlows = actionFlows
		conj.metricFlows = metricFlows
```

Reply:
Ok.